patches/@mswjs+interceptors+0.25.16.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@mswjs/interceptors/lib/node/interceptors/fetch/index.mjs b/node_modules/@mswjs/interceptors/lib/node/interceptors/fetch/index.mjs
+index 717678e..6790eb0 100644
+--- a/node_modules/@mswjs/interceptors/lib/node/interceptors/fetch/index.mjs
++++ b/node_modules/@mswjs/interceptors/lib/node/interceptors/fetch/index.mjs
+@@ -122,7 +122,7 @@ var _FetchInterceptor = class extends Interceptor {
+         return mockedResponse;
+       }
+       this.logger.info("no mocked response received!");
+-      return pureFetch(request).then((response) => {
++      return pureFetch(resolvedInput, init).then((response) => {
```

Comment:
Interesting. The `request` is created by taking `init` into account:

https://github.com/mswjs/interceptors/blob/b1bc5a5adf31726996d0935aa4b178eade1f9ac5/src/interceptors/fetch/index.ts#L50

Can you tell me more about how you reproduce this issue? Would love to make a regression test out of it.

Reply:
Something with happy-dom when it's creating the url object for the request. seems like something doesn't get serialized as i debugged it was calling the URL constructor with the _string_ `[object Request]` resulting in a `Invalid URL` error